### PR TITLE
Apply `stylish-haskell` to project

### PR DESCRIPTION
**Pull request by newhoggy**
# Changelog

```yaml
- description: |
    Apply `stylish-haskell` to project
  compatibility: no-api-changes
  type: maintenance
```

# Context

PRs tend to be a lot bigger than they need to be because`stylish-haskell` rules as to fix imports and the code is full of imports that haven't been fixed yet.

We only use `stylish-haskell` to format imports.  It's probably better to get all the pain out of the way by applying the `stylish-haskell` rules across the entire project so that future PRs can be smaller.

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] The change log section in the PR description has been filled in
- [ ] New tests are added if needed and existing tests are updated.  These may include:
  - golden tests
  - property tests
  - roundtrip tests
  - integration tests
  See [Runnings tests](https://github.com/input-output-hk/cardano-node-wiki/wiki/Running-tests) for more details
- [ ] Any changes are noted in the `CHANGELOG.md` for affected package
- [ ] The version bounds in `.cabal` files are updated
- [ ] CI passes. See note on CI.  The following CI checks are required:
  - [ ] Code is linted with `hlint`.  See `.github/workflows/check-hlint.yml` to get the `hlint` version
  - [ ] Code is formatted with `stylish-haskell`.  See `.github/workflows/stylish-haskell.yml` to get the `stylish-haskell` version
  - [ ] Code builds on Linux, MacOS and Windows for `ghc-8.10.7` and `ghc-9.2.7`
- [ ] Self-reviewed the diff

# Note on CI
If your PR is from a fork, the necessary CI jobs won't trigger automatically for security reasons.
You will need to get someone with write privileges.  Please contact IOG node developers to do this
for you.

### CODE REVIEW

**Review by carbolymer — APPROVED** [review]
Thanks for that! I think you can make this job in the stylish workflow mandatory now: https://github.com/input-output-hk/cardano-cli/actions/runs/5508573193/workflow?pr=69#L51 (can be a part of this pr)
EDIT: or not, there seem to be some issues with CPP extension: https://github.com/haskell/stylish-haskell/issues/178

File: cardano-cli/app/cardano-cli.hs [hunk 1]
```diff
@@ -5,19 +5,16 @@
...
 import           Cardano.CLI.Environment (getEnvCli)
 import           Cardano.CLI.Parsers (opts, pref)
 import           Cardano.CLI.Run (renderClientCommandError, runClientCommand)
 import           Cardano.CLI.TopHandler
 import qualified Cardano.Crypto.Init as Crypto
-#ifdef UNIX
-import           System.Posix.Files
-#endif
 
+import           Control.Monad.Trans.Except.Exit (orDie)
```
Comment (Jimbo4350) [inline cardano-cli/app/cardano-cli.hs]: Doesn't this break the `#ifdef` statement?

Comment (carbolymer) [reply cardano-cli/app/cardano-cli.hs]: Good catch! Seems like a known issue with stylish haskell 😞 https://github.com/haskell/stylish-haskell/issues/178

Comment (newhoggy) [reply cardano-cli/app/cardano-cli.hs]: Good point.  We can work around this with a proxy module.

Comment (newhoggy) [reply cardano-cli/app/cardano-cli.hs]: Files that contain CPP will be skipped in the `stylish-haskell` check.

**Review by carbolymer — APPROVED** [review]

**Pull request merged**